src/main/java/chess/domain/board/Board.java

```diff
@@ -0,0 +1,191 @@
+package chess.domain.board;
+
+import chess.domain.piece.Bishop;
+import chess.domain.piece.Camp;
+import chess.domain.piece.Empty;
+import chess.domain.piece.King;
+import chess.domain.piece.Knight;
+import chess.domain.piece.Pawn;
+import chess.domain.piece.Piece;
+import chess.domain.piece.Queen;
+import chess.domain.piece.Role;
+import chess.domain.piece.Rook;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Board {
+    private static final boolean IS_MOVED = true;
+    private static final int BOARD_LINE_SIZE = 8;
+
+    private final Map<Square, Piece> board;
+
+    public Board() {
+        this.board = generateBoard();
+    }
+
+    private LinkedHashMap<Square, Piece> generateBoard() {
+        final LinkedHashMap<Square, Piece> board = new LinkedHashMap<>();
```

Comment:
```suggestion
    private Map<Square, Piece> generateBoard() {
        final Map<Square, Piece> board = new LinkedHashMap<>();
```

인터페이스에 대고 프로그래밍하기 라는 규칙이 있습니다 :)
https://javacan.tistory.com/entry/OO-Basic-3-Flexibility

Reply:
`구현체가 아닌 인터페이스에 대고 프로그래밍`
좋은 아티클 추천 감사합니다 ˙ᵕ˙ 
해당 리뷰를 저번에도 받은 적이 있는데 놓쳤던 부분인 것 같습니다. 🥲 수정하였습니다 !